test/integration/exceptionalOpeningTimes.js

```diff
@@ -65,20 +82,27 @@ describe('app', () => {
 
           done();
         });
+    });
+  });
+
+  describe('opening times with no exceptional opening times', () => {
+    before(() => {
+      tk.travel(mockCurrentDate2());
+    });
+
+    after(() => {
       tk.reset();
     });
-    it('should return reception and surgery opening times for the date but not the ones in the past', (done) => {
-      tk.travel(mockCurrentDate1());
+    it('should not return any changed info', (done) => {
```

Comment:
Description could be more specific, i.e. should not display any changes to opening times